src/main/java/bio/terra/service/configuration/ConfigurationService.java

```diff
@@ -227,6 +229,11 @@ private void setConfiguration() {
             ConfigFaultCountedModel.RateStyleEnum.FIXED);
         addFaultSimple(FILE_DELETE_LOCK_CONFLICT_CONTINUE_FAULT);
 
+        // File delete lock faults. These are used by DatasetConnectedTest > testThatRetryFails
```

Comment:
Maybe copy/paste on the comment? I think these are dataset lock faults, not file delete lock faults.